### ISSUE TITLE
How to encode N frames only?

### ISSUE THREAD

**Issue by shubhamchdhary** [gpac/gpac#2494]
Hi, I want to capture and encode only N frames (say 10). I'm using the below command
```
gpac -i video://:fmt=x11grab:framerate=30:video_size=1280x720:dev=: enc:c=libx265 reframer:frames=1,2,3,4,5,6,7,8,9,10 -o out.mp4
```
This saves only 10 frames, but the command keeps running on the terminal. Ideally, it should stop once 10 frames are encoded and saved. I also tried with `xs` and `xe` options, but they also didn't help. The command works absolutely fine with a file as input but it isn't working with screen capture. How to do this?

**Comment by jeanlf** [comment]
Indeed, this was not coded (stop sending for reframer:frames option).
Now on master, thanks for the report.

**Comment by shubhamchdhary** [comment]
Okay, thanks.

**Comment by shubhamchdhary** [comment]
> Indeed, this was not coded (stop sending for reframer:frames option). Now on master, thanks for the report.

Hi @jeanlf, I've compiled GPAC's latest master branch after your fix. It does stop after encoding 15 frames. But fails to generate a working file and throws this warning

```
[MP4Mux] Session aborted before writing to file, use fragmented storage mode to record session
```

For this when I use `frag` option it leads to this problem https://github.com/gpac/gpac/issues/2492#issue-1756454308 that I raised an issue about already. Moreover, `flat` option also doesn't work and throws the same warning (use fragmented mode).
